modules/item/view.php

```diff
@@ -17,17 +17,32 @@
 
 $itemID = $params->get('id');
 
-$col  = 'items.id AS item_id, name_english AS identifier, ';
-$col .= 'name_japanese AS name, type, ';
-$col .= 'price_buy, price_sell, weight/10 AS weight, defence, `range`, slots, ';
-$col .= 'equip_jobs, equip_upper, equip_genders, equip_locations, ';
-$col .= 'weapon_level, equip_level AS equip_level_min, refineable, view, script, ';
-$col .= 'equip_script, unequip_script, origin_table, ';
+$job_list = array_keys(Flux::config('EquipJobs')->toArray());
+$class_list = array_keys(Flux::config('EquipUpper')->toArray());
+$equip_list = array_keys(Flux::config('EquipLocations')->toArray());
+$trade_list = array_keys(Flux::config('TradeRestriction')->toArray());
+
+if(!$server->isRenewal) {
+	array_splice($job_list, 25);
+	array_splice($class_list, 4);
```

Comment:
Looks weird, maybe we can do  this in some other way?

Reply:
This is a draft of the code.
Maybe better to split the structure of jobs and classes (in config folder) into subarrays and get it depending on the type of server mechanics like:
```
'prerenewal' => array( list of possible jobs ),
'renewal' => array( list of possible jobs )
```
but I have not yet figured out how to do it better

Reply:
Maybe we can split it like this, but merge it somehow while loading jobs list from file.

Like new function in `lib/Flux/Flux.php`
```php
public static function getEquipJobs()
{
	$EquipJobs = Flux::config('EquipJobs')->toArray();
         if($isRenewal) { ... }
	return $EquipJobs;
}
```